receiver/varnishreceiver/factory.go

```diff
@@ -22,23 +22,21 @@ import (
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/receiver"
 	"go.opentelemetry.io/collector/receiver/scraperhelper"
-)
 
-const (
-	typeStr = "varnish"
+	"github.com/GoogleCloudPlatform/opentelemetry-operations-collector/receiver/varnishreceiver/internal/metadata"
 )
 
 // NewFactory create a factory for the varnish receiver.
 func NewFactory() receiver.Factory {
 	return receiver.NewFactory(
-		typeStr,
+		metadata.Type,
 		createDefaultConfig,
 		receiver.WithMetrics(createMetricsReceiver, component.StabilityLevelBeta))
```

Comment:
Should we change `metadata.yaml` to specify the "beta" stability level, and use `metadata.StabilityLevel` here?